integration_test/third_party_apps_test.go

```diff
@@ -806,8 +806,9 @@ type test struct {
 }
 
 var defaultPlatforms = map[string]bool{
-	"debian-10":    true,
-	"windows-2019": true,
+	"debian-10":     true,
+	"windows-2019":  true,
+	"rocky-linux-9": true,
```

Comment:
Why is this in here? Did you mean to keep it or did you just do this to trigger the tests and you meant to revert it before merging?

Reply:
@martijnvans you'll have to excuse my ignorance here. I was told I needed to set it to trigger the tests. I wasn't sure at what point (or if) I needed to remove this line. I can remove it now.

Reply:
OK, that's fine, indeed without this line it would skip any 3p tests that you weren't modifying, as measured by which files you're touching. Now that tests have been run, now is a good time to remove it.

Reply:
probably want to update the PR title too.